openkfw/TruBudget
# 1861 stretch workflow details dialog

### Checklist




### Description
- Workflow Details dialog window is now stretched to fill 80% of the screen size at all times
- this change will eliminate horizontal scroll bar presence
<img width="1258" alt="Screenshot 2024-06-06 at 15 24 22" src="https://github.com/openkfw/TruBudget/assets/55734106/5ccbdae7-519a-4384-b113-3ca04423cb33">


### How to test

1. login
2. go to projects -> sub projects -> workflow items
3. select info icon in workflow item and see that dialog window is stretched

Closes #1861
